### ISSUE TITLE
dockerng.running port_bindings does not support "containerPort" specification

### ISSUE THREAD

**Issue by alekti** [saltstack/salt#26100]
The documentation for dockerng.running state, port_bindings property claims to support the following value type:

> containerPort - Bind an ephemeral port on all of the host's interfaces to a specific port within the container.

Accompanied with that documentation is an example which uses that syntax:

``` sls
foo:
  dockerng.running:
    - image: bar/baz:latest
    - port_bindings:
      - 5000:5000
      - 2123:2123/udp
      - 8080
```

However, when I set the property like this:

``` sls
foo:
  dockerng.running:
    - image: debian:jessie
    - port_bindings:
      - 80
```

then state fails with the comment: `Invalid port_bindings configuration. See the documentation for proper usage.`

**Comment by jfindlay** [comment]
@alekti, thanks for the report.

**Comment by epelc** [comment]
@jfindlay any chance this will be fixed? Is there a workaround?

**Comment by terminalmage** [comment]
This has been fixed since 2017.7.0. Marking as closed.
